<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPathsWidget</class>
 <widget class="QWidget" name="ProjectPathsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>627</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="KComboBox" name="projectPaths">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="addPath">
         <property name="toolTip">
          <string>Add a new sub-project path entry.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="removePath">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Delete current sub-project path entry.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="KTabWidget" name="languageParameters">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="includePathsPage">
      <attribute name="title">
       <string>Includes/Imports</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="IncludesWidget" name="includesWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>8</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="definesPage">
      <attribute name="title">
       <string>Defines</string>
      </attribute>
      <layout class="QVBoxLayout" name="gridLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="DefinesWidget" name="definesWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>8</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Compiler</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="compiler">
         <property name="toolTip">
          <string>Select a compiler that will be used to determine standart include directories and macros. </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item>
      <widget class="QPushButton" name="configureCompilers">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Configure...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="kcfg_reparse">
       <property name="toolTip">
        <string>Check this if you want the project to be reparsed to apply the changes.</string>
       </property>
       <property name="text">
        <string>Reparse the project</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="batchEdit">
       <property name="toolTip">
        <string>Edit multiply includes/defines at the same time.</string>
       </property>
       <property name="text">
        <string>&amp;Batch edit...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DefinesWidget</class>
   <extends>QWidget</extends>
   <header>defineswidget.h</header>
  </customwidget>
  <customwidget>
   <class>IncludesWidget</class>
   <extends>QWidget</extends>
   <header>includeswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
